src/models/constraint.rs

```diff
@@ -11,25 +11,40 @@ Copyright (c) 2022 Uber Technologies, Inc.
  limitations under the License.
 */
 
+use super::default_configs::{default_matcher, default_queries};
 use derive_builder::Builder;
 use getset::Getters;
+use pyo3::prelude::*;
```

Comment:
Not sure about Rust best practices, but for every other language our coding guidelines tend to be _against_ star imports.